sematic/ui/src/components/EnvironmentProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import { useAtom } from "jotai";
+import { EnvContext } from "../index";
+import { useEnv, userAtom } from "../hooks/appHooks";
+import { ReactNode } from "react";
+import Alert from "@mui/material/Alert/Alert";
+
+export default function EnvironmentProvider({children}: {children: ReactNode}) {
+    const [user] = useAtom(userAtom);
+
+    const {value, error } = useEnv(user);
```

Comment:
```suggestion
    const { value, error } = useEnv(user);
```
Also, other files have 2 spaces as indentation, this one and `appContext.ts` have 4.